pkg/es/config/config.go

```diff
@@ -63,6 +63,7 @@ type Configuration struct {
 	BulkFlushInterval     time.Duration  `mapstructure:"-"`
 	IndexPrefix           string         `mapstructure:"index_prefix"`
 	IndexDateLayout       string         `mapstructure:"index_date_layout"`
+	IndexRollover         string         `mapstructure:"-"`
```

Comment:
This is a poor name, what does "index rollover" mean? From the flag description it seems to mean "frequency", so let's name it IndexRolloverFrequency (and all other vars/flags respectively)